### ISSUE TITLE
twisted.internet.task.react documentation redundantly specifies the default value for argv

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6419]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)| @jonathanj reported|
|-|-|
|Trac ID|trac#6419|
|Type|enhancement|
|Created|2013-04-08 10:57:36Z|
|Branch|https://github.com/twisted/twisted/tree/react-argv-documentation-6419|

The twisted.internet.task.react documentation repeats the default value for `argv` without adding any real value, the latter part of the parameter documentation should rather describe the meaning of the default value.

<details><summary>Searchable metadata</summary>

```
trac-id__6419 6419
type__enhancement enhancement
reporter__jonathanj jonathanj
priority__normal normal
milestone__ 
branch__branches_react_argv_documentation_6419 branches/react-argv-documentation-6419
branch_author__jonathanj jonathanj
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1365418656000000 1365418656000000
changetime__1365428435000000 1365428435000000
version__None None
owner__jonathanj jonathanj

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_1"></a>|@jonathanj commented|
|-|-|

(In [37992]) Branching to 'react-argv-documentation-6419'

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)|@jonathanj removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)<a name="note_3"></a>|@Julian set owner to @jonathanj|
|-|-|

Sounds better. Think it's mergeable, presuming it might need a `misc` topfile or whatever while doing so.

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_4"></a>|@jonathanj set status to `closed`|
|-|-|

(In [37995]) Merge react-argv-documentation-6419

Author: jonathanj
Reviewer: Julian
Fixes: [#6419](https://github.com/twisted/twisted/issues/6419)

Describe the meaning of the default value for twisted.internet.task.react's "argv" parameter.
